src/Avalonia.Base/Input/DataFormatKind.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Avalonia.Input;
+
+/// <summary>
+/// Represents the kind of a <see cref="DataFormat"/>.
+/// </summary>
+public enum DataFormatKind
+{
+    /// <summary>
+    /// The data format is specific to the application.
+    /// </summary>
+    Application,
+
+    /// <summary>
+    /// The data format is specific to the current operating system.
+    /// </summary>
+    OperatingSystem,
```

Comment:
I feel like `Platform` would match here better, and be more in line with other Avalonia APIs where we refer to OS specific as "platform" specific.

But then it will add confusion about CrossPlatform